services/ark-cluster-memory/ark-cluster-memory/src/memory-store.ts

```diff
@@ -44,7 +44,8 @@ export class MemoryStore {
       timestamp: new Date().toISOString(),
       session_id: sessionID,
       query_id: '', // Legacy method without query_id
```

Comment:
oh weird i know that this is not your code but can you check whether we need this query id?